.github/workflows/merge-demo.yml

```diff
@@ -55,7 +55,7 @@ jobs:
             file: api/openshift.deploy.yml
             overwrite: true
             parameters:
-              -p OC_NAMESPACE=a4b31c-test
+              -p OC_NAMESPACE=${{ vars.OC_NAMESPACE }}
```

Comment:
Hi Catherine, just curious because I can't verify this. 
Is this environment variable from github? Value exists at github for test and prod I assume; and is based on 'environment' in template for different values? Why is this change necessary?

Reply:
yes, this is environment variable, I just feel we don't need to hardcode it. But it's unrelated to database restore, I just noticed it. I can change it back if we feel unsafe. But ideally we should use the variable, not hardcode it.

<img width="712" alt="Screen Shot 2024-08-20 at 9 37 59 AM" src="https://github.com/user-attachments/assets/e9cf7c1d-026e-44cb-913d-a43cac34708c">